src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -550,7 +550,7 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     manager.register_pass<ov::pass::RemoveConcatZeroDimInput>();
     manager.register_pass<ov::pass::TransposeSinkingGeneral>();
     manager.register_pass<ov::pass::ReverseShapeAndTypeInfer>();
-    manager.register_pass<ov::pass::ResolveNameCollisions>();
+    manager.register_pass<ov::pass::ResolveNameCollisions>(true);
```

Comment:
please clarify,  why do we need to resolve uniqueness for non-autogenerated names. TF should provide unique names for nodes

Reply:
TF Frontend can create additional friendly names during conversion

Reply:
if it creates additional friendly names, it means they are auto-generated. For auto-generated names, this transformation already works. Please correct me if I am wrong.

Reply:
There's no guarantee this won't create duplicates if someone adds code manually setting friendly names.

Reply:
And maybe somehow it's possible to create duplicates in TF, some workaround or trick

Reply:
Can some test be added to simulate the "trick" that name will be duplicated?